sdk/test/logs/simple_log_processor_test.cc

```diff
@@ -116,8 +116,6 @@ TEST(SimpleLogProcessorTest, ShutdownCalledOnce)
   EXPECT_EQ(true, processor.Shutdown());
   EXPECT_EQ(1, num_shutdowns);
 
```

Comment:
So it's not ok to call `Shutdown` twice?

Reply:
Sorry,I missed it. It should be safe to call `Shutdown` twice and it should only call `exporter.Shutdown` only once. I have already push the commit for this check.